src/meta/http/MetaHttpReplaceHostHandler.h

```diff
@@ -51,5 +51,5 @@ class MetaHttpReplaceHostHandler : public proxygen::RequestHandler {
 }  // namespace meta
 }  // namespace nebula
 
-#endif  // META_METAHTTPReplaceHostHANDLER_H
+#endif  // META_HTTP_METAHTTPReplaceHostHANDLER_H
```

Comment:
Emmm, this comment is not right.

Reply:
Good catch. Use `#pragma once` to replace the macro. I'll update the code style to recommend to use this `#pragma`